src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/csip/CSIPResourceIntTest.kt

```diff
@@ -408,5 +409,18 @@ class CSIPResourceIntTest : IntegrationTestBase() {
         .jsonPath("decision.actions.simReferral").isEqualTo(false)
       // Check create date time?
     }
+
+    @Test
+    fun `will return CSIP Document information`() {
+      webTestClient.get().uri("/csip/${csip1.id}")
+        .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_CSIP")))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("id").isEqualTo(csip1.id)
+        .jsonPath("documents[0].fileName").isEqualTo("doc1.txt")
+        .jsonPath("documents[0].status.code").isEqualTo("PUBLIC")
+        .jsonPath("documents[0].status.description").isEqualTo("Public")
+    }
```

Comment:
I actually don't want to return documents information here  - for now this is ok but there could be multiple documents and it could return a lot of data - will remove in next pr and add to new endpoint to retrieve individual documents